examples/random/main.go

```diff
@@ -27,9 +28,10 @@ import (
 
 var (
 	addr          = flag.String("listen-address", ":8080", "The address to listen on for HTTP requests.")
-	uniformDomain = flag.Float64("random.uniform.domain", 200, "The domain for the uniform distribution.")
-	normDomain    = flag.Float64("random.exponential.domain", 200, "The domain for the normal distribution.")
-	normMean      = flag.Float64("random.exponential.mean", 10, "The mean for the normal distribution.")
+	uniformDomain = flag.Float64("uniform.domain", 200, "The domain for the uniform distribution.")
+	normDomain    = flag.Float64("exponential.domain", 200, "The domain for the normal distribution.")
+	normMean      = flag.Float64("exponential.mean", 10, "The mean for the normal distribution.")
+	diurnalPeriod = flag.Duration("diurnal-period", 10*time.Minute, "The duration of a diurnal oscillation period.")
```

Comment:
It's how long you want your day to be :) But I can give it a different name if you want...